src/amber/cli/templates/app/docker-compose.yml.ecr

```diff
@@ -15,36 +15,29 @@ services:
   app:
     build: .
     image: <%= @name %>
-    command: 'amber watch'
-    working_dir: /app/local
+    command: amber watch
     environment:
       DATABASE_URL: <%= @db_url %>
-      SMTP_URL: mail:1025
     ports:
-      - '3000:3000'
+      - 3000:3000
     links:
       - db
-      - mail
     volumes:
-      - '.:/app/local'
+    - .:/app
+    - nodes:/app/node_modules
+    - shards:/app/lib
 
   migrate:
     build: .
     image: <%= @name %>
     command: bash -c '<%= @wait_for %>amber db migrate seed'
-    working_dir: /app/local
     environment:
       DATABASE_URL: <%= @db_url %>
     volumes:
-      - '.:/app/local'
+      - .:/app
     links:
       - db
 
-  mail:
```

Comment:
I removed `mailcatcher` .  Although this is really cool, it slows down the startup of docker-compose.  I think we can document this capability instead of force it on all users since many apps will not require it.